### ISSUE TITLE
[issue name]

### ISSUE THREAD

**Issue by Masternode24** [iron-fish/ironfish#1522]
### What happened?

After 70h of work ...

```
root@web:~# ironfish deposit
yarn run v1.22.18
$ yarn build && yarn start:js deposit
$ tsc -b
$ cross-env OCLIF_TS_NODE=0 IRONFISH_DEBUG=1 node bin/run deposit
All decryptable notes should be in the noteToNullifier map
Done in 2.06s.

```

### Version

0.35

### Graffiti

Masternode24.de

**Comment by lwisne** [comment]
Still working on this error popping up from time to time.

**Comment by hughy** [comment]
Errors like this one should be resolved by the Wallet 2.0 refactor released in [v0.1.48](https://github.com/iron-fish/ironfish/releases/tag/v0.1.48)

Please open a new issue if you run into this again!
